test/unit/publishing_api_updater_test.rb

```diff
@@ -0,0 +1,28 @@
+require "test_helper"
+require "gds_api/test_helpers/publishing_api_v2"
+
+class PublishingAPIUpdaterTest < ActiveSupport::TestCase
+  include GdsApi::TestHelpers::PublishingApiV2
+
+  context ".perform(edition_id)" do
+    setup do
+      Edition.after_save.clear #avoids automated calls to PublishingAPIUpdater
```

Comment:
This is dangerous because the test compromises the implementation.

We should take an approach more like this:
https://github.com/alphagov/publisher/commit/a377f52b2a7c1f706818eefed36b0fc0430b7a14#diff-63598632bcbe0c01faf4cedced45e944R18